plugin/documents.py

```diff
@@ -545,9 +550,24 @@ def _render_code_lens(self, name: str, index: int, region: sublime.Region, comma
                 annotation = command["title"]
         else:
             annotation = "..."
-        annotation = '<div class="codelens">{}</div>'.format(annotation)
-        accent = self.view.style_for_scope("region.greenish markup.accent.codelens.lsp")["foreground"]
-        self.view.add_regions(self._code_lens_key(index), [region], "", "", 0, [annotation], accent)
+
+        show_code_lens = userprefs().show_code_lens
+        if show_code_lens == "phantom":
+            key = region.to_tuple()
+            try:
+                phantom = self._code_lens_phantom_regions[key]
+            except KeyError:
+                self._code_lens_phantom_regions[key] = phantom = CodeLensPhantom(self.view, region)
+            if annotation != "...":
+                phantom.add_annotation(annotation)
+        elif show_code_lens == "annotation":
+            annotation = '<div class="codelens">{}</div>'.format(annotation)
```

Comment:
Is `class` used for anything? And if not, there is likely no need to wrap it in a `div` also.

Reply:
This class was not mine, it was there from before I sent the PR. I do not know why it is there but I can remove it if it's unnecessary.